com.unity.render-pipelines.high-definition/Editor/Sky/HDRISky/HDRISkyEditor.cs

```diff
@@ -95,9 +95,7 @@ public override void OnEnable()
             m_ShadowTint                = Unpack(o.Find(x => x.shadowTint));
 
             m_IntensityTexture = RTHandles.Alloc(1, 1, colorFormat: GraphicsFormat.R32G32B32A32_SFloat);
-            var hdrp = HDRenderPipeline.defaultAsset;
-            if (hdrp != null)
-                m_IntegrateHDRISkyMaterial = CoreUtils.CreateEngineMaterial(hdrp.renderPipelineResources.shaders.integrateHdriSkyPS);
+            m_IntegrateHDRISkyMaterial = CoreUtils.CreateEngineMaterial(HDRenderPipelineGlobalSettings.instance.renderPipelineResources.shaders.integrateHdriSkyPS);
```

Comment:
Before we checked against null asset. It is not done anymore. The HDRenderPipelineGlobalSettings.instance can still be null?

Reply:
This seems to be from olden times where sometimes we got init problems. It's probably not an issue anymore but we never know...